test/unittest/logging/CMakeLists.txt

```diff
@@ -39,5 +39,37 @@ if(NOT ((MSVC OR MSVC_IDE) AND EPROSIMA_INSTALLER))
                 )
         endif()
         add_gtest(LogTests SOURCES ${LOGTESTS_SOURCE})
+
+        set(LOGFILETESTS_SOURCE
```

Comment:
Split in `LOGFILETESTS_TEST_SOURCE` adding only `LogFileTests.cpp` and `LOGFILETESTS_SOURCE` will add `LOGFILETESTS_TEST_SOURCE` and the rest of the sources.
Then `add_gtest` will receive only `LOGFILETESTS_TEST_SOURCE`.

For more info see `test/blackbox/CMakeLists.txt:100`